cypress/integration/tests.spec.js

```diff
@@ -348,49 +348,6 @@ describe('Miscellaneous tests', function () {
     Swal.clickConfirm()
   })
 
-  it('esc key', (done) => {
```

Comment:
revert this along with https://github.com/sweetalert2/sweetalert2/pull/2191/files#r597524197